QATCH/common/licenseManager.py

```
@@ -0,0 +1,865 @@
+        """Prepares the directory or database for use as a license manager.
+        For AVN: Attempt to open the database, create a connection and cursor.
+        For DBX: Check if remote license directory exists and create it if needed.
+
+        Returns:
+            bool: True if database/directory exists or was created successfully, False otherwise.
+        """
+        if DB_SERVER == LicenseServer.AIVENIO:
+            try:
+                self.avn.connection.ping(reconnect=True)
+                self.avn.close()
+                return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)